Makefile

```diff
@@ -60,14 +60,14 @@ MIXIN_OUT_PATH_SUFFIXES := "" "-baremetal"
 JSONNET_MANIFESTS_PATH := operations/mimir
 
 # Doc templates in use
-DOC_TEMPLATES := docs/sources/operators-guide/configure/reference-configuration-parameters/index.template
+DOC_TEMPLATES := docs/sources/mimir/operators-guide/configure/reference-configuration-parameters/index.template
```

Comment:
Got it! This is where I had problems. Thank you for the fix. :)

Reply:
Happy to help!